featurizers/fingerprints.py

```diff
@@ -1,68 +1,234 @@
-import multiprocessing as mp
 from typing import Union
 
 import numpy as np
 import pandas as pd
 import rdkit.Chem as Chem
-from rdkit.Chem.rdMolDescriptors import GetMorganFingerprintAsBitVect
 
 from base import FingerprintTransformer
 
+"""
+If during multiprocessing occurs MaybeEncodingError, first check if there isn't thrown any exception inside
+worker function! (That error isn't very informative and this tip might save you a lot of time)
+"""
 
-class MorganFingerprintAsBitVect(FingerprintTransformer):
+"""
+fp_descriptors needs to be here or inside _transform() of a specific class (cannot be defined inside the __init__() of
+that class), otherwise pickle gets angry:
+        TypeError: cannot pickle 'Boost.Python.function' object
+"""
+
+
+class MorganFingerprint(FingerprintTransformer):
     def __init__(
         self,
         radius: int = 2,
         n_bits: int = 2048,
         use_chirality: bool = False,
         use_bond_types: bool = True,
         use_features: bool = False,
+        result_type: str = "default",
         n_jobs: int = 1,
```

Comment:
Trzeba jeszcze dodać opcję `sparse: bool = False`, bo każdy fingerprint może być jako macierz rzadka

Reply:
Dodaję to do init dla każdego fingerprinta. czy tak może być?

Reply:
Tak. Przy czym w przypadku MACCS przyda się ostrzeżenie w dokumentacji, że to może być średnio przydatne

Reply:
Dodałem w komentarzu i dodamy w dokumentacji